backend/admin/testdata/go/dischema/go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/cenkalti/backoff/v5 v5.0.2 // indirect
 	github.com/danieljoos/wincred v1.2.2 // indirect
 	github.com/deckarep/golang-set/v2 v2.8.0 // indirect
+	github.com/fsnotify/fsnotify v1.7.0 // indirect
```

Comment:
Unfortunately this package is buggy. It has file descriptor limit issues, and is generally just flaky.